actions/command.js

```diff
@@ -86,7 +87,7 @@ export function executeCommand(message, args) {
             break;
         }
         case '/settings':
-            dispatch(openModal({ModalId: ModalIdentifiers.USER_SETTINGS, dialogType: UserSettingsModal}));
+            dispatch(openModal({modalId: ModalIdentifiers.USER_SETTINGS, dialogType: UserSettingsModal}));
```

Comment:
oh boy, this has been broken for a good while :P

Reply:
Looks like there are a bunch of places where we use `ModalId` instead of `modalId`

Reply:
yeah, there are some places where this error occurs